### DISCUSSION TITLE
PV mount to kafka connect

### DISCUSSION THREAD

**Discussion by mohan-nagandlla** [strimzi/strimzi-kafka-operator#5755] [General]
HI team i just want to add the Persistent Volume mount to strimzi kafka conncet is it possible?

**Comment by scholzj** [comment]
No, Strimzi does not support that currently.

**Reply by mohan-nagandlla** [reply]
ok thank you

**Comment by rvnhere** [comment]
Hello @scholzj is there to a plan to make this feature available in the future ? My use case is to use FilePulse connector to stream XML files (created by another application) located on a NFS drive to Kafka (managed by Strimzi). But to make the NFS location available for Kafka Connect, I need to be able to mount a PV. Are there any other workarounds for this ?

**Reply by scholzj** [reply]
I think there is some enhancement issue covering this. In general, I'm not aware of anyone planning to work on anything like this. It also needs to be properly thoughts through how it can / should be done (for example the whole setup with RWX volumes such as NFS versus RWO volumes etc.). There seems to be some demand, so if someone wants to contribute it, why not. But this probably should have a proposal first.
